MSAL/src/public/MSALWebviewSelection.h

```diff
@@ -22,21 +20,24 @@
 // LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 // THE SOFTWARE.
-//
-//------------------------------------------------------------------------------
-
-#import <Foundation/Foundation.h>
-
-typedef void (^MSALWebUICompletionBlock)(NSURL *response, NSError *error);
-
-@interface MSALWebUI : NSObject
-
-+ (void)startWebUIWithURL:(NSURL *)url
-                  context:(id<MSALRequestContext>)context
-          completionBlock:(MSALWebUICompletionBlock)completionBlock;
 
-+ (BOOL)handleResponse:(NSURL *)url;
+#ifndef MSALWebviewSelection_h
+#define MSALWebviewSelection_h
 
-+ (BOOL)cancelCurrentWebAuthSession;
+typedef NS_ENUM(NSInteger, MSALWebviewSelection)
+{
+    // Uses WKWebView
+    MSALWebviewSelectionEmbedded,
```

Comment:
let's update this to be clear on what the choice is (as we discussed, call it WKWebView, Session etc), if we can get agreement on that

Reply:
Yes, done